content/02.introduction.md

```diff
@@ -19,17 +19,25 @@ This workflow provides a strong and clear advantage over sending files back-and-
 Through its combination of version control and collaborative features, GitHub facilitates open-source code alongside collaborative development [@doi:10.1371/journal.pcbi.1004947].
 
 Git is the version control system that enables all the collaborative tools available on GitHub.
-Git was originally developed as a fast, lightweight and open-source system that would allow software engineers to efficiently develop and collaborate in projects [@url:https://git-scm.com/book/en/v2/Getting-Started-A-Short-History-of-Git#:~:text=A%20Short%20History%20of%20Git%20As%20with%20many,were%20passed%20around%20as%20patches%20and%20archived%20files.]. 
+Git was originally developed as a fast, lightweight and open-source system that would allow software engineers to efficiently develop and collaborate in projects [@url:https://git-scm.com/book/en/v2/Getting-Started-A-Short-History-of-Git]. 
 Since its launch in 2005, Git has become the leading version control system in software development and in other disciplines that similarly require collaboration and community contributions, such as scientific research [@doi:10.1109/MS.2012.61].
 Although the understanding of basic concepts of Git (such as commit, push, pull, checkout; see [Box 1](#definitions)) is necessary, the GitHub web-based platform and its integrated development environments (such as GitHub Desktop) allow users to perform most repository and data management operations without opening Git command-line sessions, making these functionalities available to users who are less familiar with software development.
 
+Version control involves tracking the state of the files in a folder, called a "repository" (see [Box 1](#definitions)), through the evolution of a project.
+A basic workflow using git and GitHub is to 1) make changes to files in a repository on your computer, 2) record related changes in files as a "commit" (see [Box 1](#definitions)) along with a message describing the changes, 3) synchronize commits with GitHub (see "push" in [Box 1](#definitions)) so that the repository on the web is up to date. 
+Collaborators (or you on a different computer) can then view your changes online and synchronize them to their own computers (see "pull" in [Box 1](#definitions)).
+Commits act like snapshots and one can revisit the state of the project to any commit to view or undo changes.
+If the modified files are plain text (e.g. computer code or .txt files) only the differences from the previous commit are recorded, allowing you frequently commit changes without causing the size of the project to grow excessively.
```

Comment:
add a comma and "to": If the modified files are plain text (e.g. computer code or .txt files), only the differences from the previous commit are recorded, allowing you to frequently commit ...